animation_nodes/nodes/mesh/set_vertex_weights.py

```diff
@@ -0,0 +1,131 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... data_structures import VirtualDoubleList
+from ... base_types import AnimationNode, VectorizedSocket
+
+modeItems = [
+    ("ALL", "All", "Set weight of every vertex", "NONE", 0),
+    ("INDEX", "Index", "Set weight of a specific vertex", "NONE", 1)
+]
+
+groupIdentifierTypeItems = [
+    ("INDEX", "Index", "Get vertex group based on the index", "NONE", 0),
+    ("NAME", "Name", "Get vertex group based on the name", "NONE", 1)
+]
+
+groupNotFoundMessage = "group not found"
+noMeshMessage = "no mesh object"
+editMeshMessage = "object is not in object mode"
```

Comment:
Now that we grouped the exceptions together, there is no need to define those as variables, just put them in the function call. Also, capitalize the first letter and add a period at the end.

Reply:
Removed the error variables.